kokoro/gcp_ubuntu/kokoro_build_presubmit.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
There seems to be a bit of duplication going on. Would it make sense to move some of it to a common library?

Reply:
Was able to get the Kokoro job name from the sys. variable and create a unified script.